### ISSUE TITLE
need add pinax.images as dependence in INSTALLED_APPS (write it to document,pls)

### ISSUE THREAD

**Issue by ruandao** [pinax/pinax-blog#99]
otherwise when run `./manage.py migrate` this error happen
```
RuntimeError: Model class pinax.images.models.ImageSet doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

**Comment by grahamu** [comment]
@ruandao I believe pinax-blog v6.0.2 release fixes this issue with documentation indicating the need to add pinax-images to SETTINGS. Please close this issue if so, thanks!

https://github.com/pinax/pinax-blog/releases/tag/v6.0.2

**Comment by pjmcdermott** [comment]
Just came across this issue with `pinax-blog==6.3.1`:

```
RuntimeError: Model class pinax.images.models.ImageSet doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```
```
$ pip freeze | grep pinax
pinax-blog==6.3.1
pinax-images==2.1.0
```
`pinax.images` in `INSTALLED_APPS` resolves. The requirement is not mentioned at [pinax/pinax-blog](https://github.com/pinax/pinax-blog), which is what I was following.

**Comment by pjmcdermott** [comment]
I'm a bit confused why `pip freeze` is showing `pinax-blog==6.3.1` when the latest release on github is 6.1.1.... What have I done??

**Comment by michaelkrukov** [comment]
Adding `pinax.images` to INSTALLED_APPS fixed issue for me.
```
RuntimeError: Model class pinax.images.models.ImageSet doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```
Django==2.0
pinax-blog==7.0.1
pinax-images==3.0.1

**Comment by grahamu** [comment]
@pjmcdermott and @michaelkrukov thanks for bringing this up. Although I did fix the original issue via https://github.com/pinax/pinax-blog/commit/a36ba49ac6f28c12e1b336c8a05dcab7dae61b5b, I subsequently did not copy over the same guidance when converting documentation to single README.md file.

My apologies. This oversight is now fixed in 7.0.2, just published, with updated documentation, testing requirements and configuration. Added test proving pinax-images is required for mgmt create post view when using default templates.

`pinax-images` _is_ a requirement for `pinax-blog`
